### ISSUE TITLE
Learning Rate constant ( Not increasing)

### ISSUE THREAD

**Issue by Shivakishore14** [datitran/raccoon_dataset#9]
Hi, I tired to train the raccoon detector locally, with the tfrecords provided in your repo, but for some reason the learning rate is not increasing and I even tried exporting the check-points and perform detection on an image from the training dataset it also dint work. any idea on how to solve this issue
info
OS: windows 10
tensorflow version : 1.2
![capture](https://user-images.githubusercontent.com/13432471/29654027-2b8afb2c-88c9-11e7-94d5-1530a66fcd9e.PNG)

**--UPDATE--**
After letting it train for 4Hrs it was able to detect raccoons in images but still the learning rate is constant. Any idea why its like that?
And i was not able to even detect objects in my custom dataset
https://stackoverflow.com/questions/45546556/tensorflow-object-detection-not-learning-on-custom-dataset-monkey-images

**Comment by datitran** [comment]
The learning rate is fine. It's only increasing if you set it up the config file. I've checked the post on Stackoverflow. Will download your dataset tomorrow and try it from my side.
